src/shared/components/case-viewer/services/case.resolver.ts

```diff
@@ -87,6 +90,10 @@ export class CaseResolver implements Resolve<CaseView> {
   private checkAuthorizationError(error: any) {
     // TODO Should be logged to remote logging infrastructure
     console.error(error);
+    if (this.onErrorCaseList && error.status === 404) {
+      this.router.navigate(['/list/case'])
```

Comment:
what error are we showing on the workbasket?
does that show as a red banner?

Reply:
as per ticket it's a green confirmation bar on top. Event trigger has these messages already.
https://github.com/hmcts/ccd-case-ui-toolkit/blob/RDM-5031/src/shared/components/case-viewer/case-event-trigger/case-event-trigger.component.ts#L78-L84

Reply:
👍